packages/payment-detection/src/erc20/escrow-info-retriever.ts

```diff
@@ -0,0 +1,133 @@
+import { PaymentTypes } from '@requestnetwork/types';
+import { ethers } from 'ethers';
+import { getDefaultProvider } from '../provider';
+import { parseLogArgs } from '../utils';
+
+// The ERC20 escrow smart contract ABI fragment containing escrow specific events.
+const erc20EscrowContractAbiFragment = [
+  'event RequestFrozen(bytes indexed paymentReference)',
+  'event InitiatedEmergencyClaim(bytes indexed paymentReference)',
+  'event RevertedEmergencyClaim(bytes indexed paymentReference)',
+];
```

Comment:
Is it possible to import the [ABI](https://github.com/RequestNetwork/requestNetwork/blob/master/packages/smart-contracts/src/lib/artifacts/ERC20EscrowToPay/0.1.0.json) directly?

Reply:
Yes, I saw that it was done in one of the other `info.retriever.ts` files.